op-program/client/l2/engineapi/block_processor.go

```diff
@@ -110,6 +110,7 @@ func NewBlockProcessorFromHeader(provider BlockDataProvider, h *types.Header) (*
 				excessBlobGas = eip4844.CalcExcessBlobGas(0, 0)
 			}
 			header.ExcessBlobGas = &excessBlobGas
+			vmenv.Context.BlobBaseFee = eip4844.CalcBlobFee(*header.ExcessBlobGas)
```

Comment:
This change should be unnecessary after https://github.com/QuarkChain/optimism/pull/19 is merged.